web-marketplace/src/components/organisms/ProjectTopSection/ProjectTopSection.tsx

```diff
@@ -93,7 +93,8 @@ function ProjectTopSection({
   );
   const creditTypeSanity = sanityCreditTypeData?.allCreditType?.find(
     creditType =>
-      creditType.name?.toLowerCase() === creditTypeData?.creditType?.name,
+      creditType.name?.toLowerCase() ===
+      creditTypeData?.creditType?.name?.toLocaleLowerCase(),
```

Comment:
why not just use `toLowerCase` here too?

Reply:
it's a mistake, thanks.